shared/src/profileData.ts

```diff
@@ -94,6 +94,7 @@ export interface ProfileData extends IndustrySpecificData {
   readonly nexusDbaName: string;
   readonly needsNexusDbaName: boolean;
   readonly operatingPhase: OperatingPhaseId;
+  readonly isNonprofitOnboardingRadio: boolean;
```

Comment:
The reason for this name is to be overly specific that this field does NOT represent if a user `isNonprofit`

They can change their legal structure at any point later, in the task itself. If they do that, their legal structure will be `nonprofit` but this would still be set to `false` if they selected "No" during onboarding.

So this is named this way to show that this field should NOT be used to logically check if a user is a nonprofit - it ONLY represents their radio selection during onboarding